vortex-proto/proto/scalar.proto

```diff
@@ -21,6 +21,7 @@ message ScalarValue {
     uint64 uint64_value = 4;
     float f32_value = 5;
     double f64_value = 6;
+    bytes decimal_value = 10;
```

Comment:
Why is this not bytes type?

Reply:
That's the current behavior, but if you look at the TryFrom impl for pb::ScalarValue we lose the type information and always cast it back into InnerScalarValue::Bytes, and then you get a Scalar(dtype = decimal, inner = Bytes) which is a broken state

Reply:
Ah I see, this is handled in `Decimal::as_scalar`

Reply:
Reverted the proto changes